.cirrus.yml

```diff
@@ -140,6 +140,10 @@ gating_task:
         - '/usr/local/bin/entrypoint.sh clean podman BUILDTAGS="exclude_graphdriver_devicemapper selinux seccomp"'
         - '/usr/local/bin/entrypoint.sh clean podman-remote-darwin'
 
+    subcommand_manpages_script:
+        # N/B: Requires podman command has been built and can be found in $PATH
+        - 'cd /go/src/github.com/containers/libpod && ./hack/podman-commands.sh'
```

Comment:
Should we reorder this so that the last Podman build check about the the normal 'clean podman' one so we have all the commands?

Reply:
I tried that at first but it didn't work for some reason :confounded:   

does it need to be 'podman clean'?

Reply:
okay, I think I figured this out.